index.html

```diff
@@ -213,5 +213,60 @@ <h3>Skills</h3>
             </div>
             </div> 
     </section>
+    <section class="contact">
+        <img class="logo" src="./img/image_geometry_1.png" alt="Logo">
+        <div class="contact-form">
+        <h2 id="contact">Get Started</h2>
+        <p>I'm always interested in hearing
+            about new projects, so if you'd like
+             to chat please get in touch.</p>
+        <form action="https://formspree.io/f/xqkonpny" method="post">
+            <div class="name-label">
+                <p>Your name</p>
+                <label for="name">
+                    <input type="text" name="name" placeholder="Enter your name" required>
+                </label>
+            </div>
+            <div class="name-label">
+                <p>Your email</p>
+                <label for="email">
+                    <input type="email" name="email" placeholder="Enter your email" required>
+                </label>
+              </div>
+              <div class="name-label">
+                  <p> Your message</p>
+                    <label for="Message">
+                        <textarea name="Message" id="message" placeholder="Enter your text here..." cols="30" rows="10" >
+                        </textarea>
```

Comment:
Nice job integrating `formspre` with your form👍. Just as required in this milestone, validation should be added to the name input field that stops it from accepting more than 30 characters😉. kindly add that to your form👍.

<img width="494" alt="image" src="https://user-images.githubusercontent.com/64643995/235886408-89a0ddc2-8617-4cdd-8e1b-714834063c84.png">

Take note from the image above that you are also required to add a validation to your text field that stops a user from submitting an empty form or submitting the form if that field is empty😉 Kindly fix that👍.